src/bench/bench.h

```diff
@@ -18,16 +18,18 @@
 /*
  * Usage:
 
-static void CODE_TO_TIME(benchmark::Bench& bench)
+static void CodeToTime(benchmark::Bench& bench)
 {
     ... do any setup needed...
-    nanobench::Config().run([&] {
+
+    bench.run([&] {
```

Comment:
Could mention options here if reviewers think it would be helpful.

```cpp
bench.run([&] {

...or with options...

bench.batch(NUM).epochs(NUM).epochIterations(NUM).unit(STRING).run([&] {
```

Reply:
I do think this would be helpful to document here

Reply:
Thanks for the feedback!  Done, and fixed up the help printed by `src/bench/bench_bitcoin -?`